src/app/(post-login)/submission/components/ProgramList.tsx

```diff
@@ -18,27 +18,14 @@
  */
 'use client';
 
-import { useTheme } from '@/lib/emotion';
-import { Table, Typography, css } from '@icgc-argo/uikit';
-import { columns } from '../tableConfig';
-
-export type ProgramsData = {
-	shortName: string;
-	name: string | null;
-	cancerTypes: Array<string>;
-	countries: Array<string> | null;
-	membershipType: ArgoMembershipKey;
-	genomicDonors: number | null;
-	submittedDonors: number;
-	commitmentDonors: number;
-	administrators: { firstName: string; lastName: string; email: string }[];
-	donorPercentage: number;
-};
+import { css } from '@/lib/emotion';
+import { Table, Typography } from '@icgc-argo/uikit';
+import { columns } from '../program-table/config';
 
+export type ProgramData = any;
```

Comment:
assuming this will be better defined later on. right?
I'd add a TODO comment, just to be sure to round back to it, but I'm not asking you to.

Reply:
Yes I'll add types in the data ticket.
I will avoid the addition of TODO because they get lost and I'm tackling it right after this feedback :)